.github/workflows/numpy2_compatibility.yml

```diff
@@ -4,6 +4,10 @@
 name: Numpy2 compatibility
 
 on:
+    # for testing
+    pull_request:
+        branches:
+        -   '*'
```

Comment:
to remove before merge

Reply:
Just added to make sure that this workflow would work. Otherwise we would have to merge the PR and then trigger manually to check if the tests pass.

The plan is to remove this before merging